mlem/contrib/pandas.py

```diff
@@ -686,4 +690,8 @@ def load_obj(cls, location: Location, modifier: Optional[str], **kwargs):
         read_args = fmt.read_args or {}
         read_args.update(kwargs)
         with location.open("rb") as f:
-            return fmt.read_func(f, **read_args)
+            df = fmt.read_func(f, **read_args)
+
+            # note: should consider using head() here, more meaningful for small dfs and vectors
+            logger.debug("Loaded dataframe object, showing 'describe'\n %s", df.head())
```

Comment:
So, `head` or `describe`? Can be both, no problem to me since it's in debug output.

Reply:
I don't know if there's a 1-size fits all honestly. I've tried both, decided to go with describe to be more friendly for large dataframes, but whatever you think @iterative/mlem  (or of course we can scratch this if you have a better idea / experience of debugging user-issues with mlem

Reply:
Can we make this even lazier? I mean df.head is still executed even if log level is info. maybe there is something like

```python
a = lazy_str(df.head)
str(a) == ... # actual execution happens here
```

Reply:
There is a truncation applied to columns by default:
<img width="1536" alt="image" src="https://user-images.githubusercontent.com/6797716/200810978-07916cbc-e68d-47bc-8086-b22f04f9854a.png">

Reply:
Just tested it with mnist as well 😄 it's safe (truncates columns like @aguschin mentioned). I think this should be controllable via pandas display options, the default is adaptive to display/terminal size